mzLib/Proteomics/ProteolyticDigestion/ProteolyticPeptide.cs

```diff
@@ -83,5 +84,22 @@ internal IEnumerable<PeptideWithSetModifications> GetModifiedPeptides(List<Modif
                 DictionaryPool.Return(twoBasedPossibleVariableAndLocalizeableModifications);
             }
         }
+
+        internal long numPossibleProteoforms(List<Modification> allKnownFixedModifications,
+        DigestionParams digestionParams, List<Modification> variableModifications)
+
```

Comment:
Instead of taking digestion params, it can just take an int paramater called "MaxNumberOfModifiedSites". DigestionParams.MaxModsForPeptide() and DigestionParams.MaxMods are equal